src/sentry/incidents/models/alert_rule.py

```diff
@@ -355,6 +355,9 @@ def subscribe_projects(
         # On activated subscription, additional query parameters will be added to the constructed query in Snuba
         created_subscriptions = []
         if self.monitor_type == monitor_type.value:
+            # TODO: remove projects from QuerySubscriptions
+            # This means we will no longer need to utilize `bulk_create_snuba_subscriptions` as there's no value
+            # in iterating over projects to call `create_snuba_subscription` per project
```

Comment:
Hmm, why are we removing projects from `QuerySubscriptions`?

Remember that the subscription system doesn't know the other systems exist and is generic. If it doesn't know which subscription a project belongs to, it doesn't have enough information to recreate a subscription.

Reply:
🤔 fair point.
Yea - i guess we could either pass `project` wherever we're "create_subscription_in_snuba"ing (whichcurrently pulls projects out of the subscription instance)
OR - we maintain duplicate references to the project in QuerySubscription and AlertRule - referencing the subscription project _only_ for snuba subscription creation, while referencing AlertRule project for everything else

Ok - i can remove this comment with a caution note on the duplication 👀